authority-portal-backend/authority-portal-quarkus/src/main/kotlin/de/sovity/authorityportal/web/services/ConnectorService.kt

```diff
@@ -171,7 +172,7 @@ class ConnectorService(
         organizationId: String,
         environment: String,
         clientId: String,
-        connector: CreateConnectorRequest,
+        connector: ConnectorCreationData,
```

Comment:
Comment: `CreateConnectorParams` var name "createParams", compared to "createRequest"